### PR TITLE
TS-refactor-styles-in-Lessons-and-Teachers-pages

### PR DESCRIPTION
**Pull request by Yashchemskyi-Oleksandr**

### CODE REVIEW

File: frontend/src/components/TeachersPage/TeachersList/TeachersList.scss [hunk 1]
```diff
@@ -1,78 +1,35 @@
...
-            justify-content: center;
+            width: 300px;
         }
     }
 
     .send-button {
-        width: 100%;
+        width: $full-size;
 
         @media (max-width: $md-small) {
```
Comment (Niknikson) [inline frontend/src/components/TeachersPage/TeachersList/TeachersList.scss]: can you use $md-s ?

File: frontend/src/components/TeachersPage/TeachersList/TeachersList.scss [hunk 2]
```diff
@@ -82,83 +39,32 @@
     @media (max-width: $md-width-lanscape) {
         position: relative;
         margin-bottom: 20px;
-        max-width: 90%;
     }
 
     @media (max-width: $md-small) {
```
Comment (Niknikson) [inline frontend/src/components/TeachersPage/TeachersList/TeachersList.scss]: can you use $md-s ?

**Review by Niknikson — APPROVED** [review]

File: frontend/src/actions/teachers.js [hunk 3]
```diff
@@ -1,129 +1,95 @@
...
+    teacher,
+});
+
+export const addTeacherStart = (teacher) => ({
+    type: actionsType.ADD_TEACHER_START,
+    teacher,
+});
+
+export const setTeacher = (teacher) => ({
+    //
```
Comment (ViktoriiaKosovan) [inline frontend/src/actions/teachers.js]: delete it and maybe use one style for action payload, e.x., not "result: teachers" - only teachers,

File: frontend/src/components/LessonsPage/LessonPage.js [hunk 4]
```diff
@@ -87,7 +87,7 @@ const LessonPage = (props) => {
             setUniqueError(true);
             return;
         }
-        handleLesson(cardObj, groupId);
+        handleLesson({ values: cardObj, groupId });
```
Comment (ViktoriiaKosovan) [inline frontend/src/components/LessonsPage/LessonPage.js]: maybe rename cardObj to values

File: frontend/src/components/LessonsPage/LessonPage.js [hunk 5]
```diff
@@ -109,7 +109,7 @@ const LessonPage = (props) => {
         const { group: copiedGroup, lesson } = params;
         setIsOpenCopyLessonDialog(false);
         if (!isNil(copiedGroup)) {
-            copyLessonCard(copiedGroup, lesson);
+            copyLessonCardStart({ group: copiedGroup, lesson });
```
Comment (ViktoriiaKosovan) [inline frontend/src/components/LessonsPage/LessonPage.js:112]: as above, maybe use group directly

File: frontend/src/sagas/teachers.js [hunk 6]
```diff
@@ -78,7 +79,7 @@ export function* removeTeacher({ id }) {
     }
 }
 
-export function* createTeacher(teacher) {
+export function* createTeacher({ teacher }) {
     const results = { ...teacher };
     if (isObjectEmpty(teacher.department) || !teacher.department?.id) {
```
Comment (ViktoriiaKosovan) [inline frontend/src/sagas/teachers.js]: Could you use isEmpty from lodash in this case?

**Review by ViktoriiaKosovan — APPROVED** [review]

File: frontend/src/sagas/teachers.js (hunk 6 above)
Comment (Yashchemskyi-Oleksandr) [reply frontend/src/sagas/teachers.js]: done

File: frontend/src/components/LessonsPage/LessonPage.js (hunk 5 above)
Comment (Yashchemskyi-Oleksandr) [reply frontend/src/components/LessonsPage/LessonPage.js:112]: done

File: frontend/src/components/LessonsPage/LessonPage.js (hunk 4 above)
Comment (Yashchemskyi-Oleksandr) [reply frontend/src/components/LessonsPage/LessonPage.js]: done

File: frontend/src/actions/teachers.js (hunk 3 above)
Comment (Yashchemskyi-Oleksandr) [reply frontend/src/actions/teachers.js]: done

**Pull request closed without merging**